README.md

```diff
@@ -1,6 +1,30 @@
 # xeus-python + JupyterLite demo
 
-https://martinrenou.github.io/xeus-python-demo/retro/notebooks/?path=demo.ipynb
+[![lite-badge](https://jupyterlite.rtfd.io/en/latest/_static/badge.svg)](https://jupyterlite.github.io/xeus-python-demo/retro/notebooks/?path=demo.ipynb)
+
+xeus-python + JupyterLite deployed as a static site to GitHub Pages, for demo purposes.
+
+## ✨ Try it in your browser ✨
+
+https://jupyterlite.github.io/xeus-python-demo/retro/notebooks/?path=demo.ipynb
+
+## How does it compare to the Pyolite kernel?
```

Comment:
Maybe we can call it the `Pyodide kernel` since this is most likely the name we'll be using after the move to https://github.com/jupyterlite/pyodide-kernel (not done yet)?

Reply:
Thanks for the review! Will do